scanners/zap/zap.py

```diff
@@ -301,6 +302,30 @@ def _setup_ajax_spider(self):
 
         self.af["jobs"].append(af_spider_ajax)
 
+    def _setup_graphql(self):
+        """Prepare a graphql job and append it to the job list"""
+
+        if self.config.get("scanners.zap.graphql", default=False) is False:
+            return
+
+        af_graphql = {
+            "name": "graphql",
+            "type": "graphql",
+            "parameters": {
+                "endpoint": self.config.get(
+                    "scanners.zap.graphql.endpoint", default=""
+                ),
+                "schemaUrl": self.config.get(
+                    "scanners.zap.graphql.schemaUrl", default=""
+                ),
+                "schemaFile": self.config.get(
+                    "scanners.zap.graphql.schemaFile", default=""
```

Comment:
if `scanners.zap.graphql.schemaFile` exists, the file needs to be imported (copied inside the container) and the path translated.
This should work (similar to `openapi.apiFile` and the `importUrlFromFile`): 
```
hostFile = self.config.get("scanners.zap.graphql.schemaFile")
if hostFile:
  contFile = os.path.join(self._container_work_dir(), "graphql.schema")
  self._include_file(host_path=hostFile, dest_in_container=contFile)
  af_graphql["parameters"]["schemaFile"] = contFile
```

if `schemaFile` does not exist or is set to False or an empty string, it will simply be skipped over.
Copying the file will crash if the file pointed does not exist, but it's probably what we want to do.